kernel/src/callback.rs

```diff
@@ -196,6 +196,10 @@ impl Callback {
         self.cb.map_or(true, |mut cb| cb.schedule(r0, r1, r2))
     }
 
+    pub fn is_some(&self) -> bool {
+        self.cb.is_some()
+    }
+
```

Comment:
I believe this is something we explicitly do not want to provide. See the discussion here: https://github.com/tock/tock/pull/2258#discussion_r542765769